docker-compose.yaml

```diff
@@ -3,32 +3,32 @@
 version: "3"
 services:
   shiori:
-    build:
-      context: .
-      dockerfile: Dockerfile.compose
+    image: ghcr.io/go-shiori/shiori
     container_name: shiori
     ports:
-      - "8080:8080"
+      - "8080:8080" #set port number if it needs to be different
```

Comment:
Please revert this to the original state, since this docker compose is meant for development purposes.